client/jetpack-onboarding/main.jsx

```diff
@@ -68,6 +70,16 @@ class JetpackOnboardingMain extends React.PureComponent {
 		} );
 	};
 
+	getSkipLinkText() {
+		const { stepName, stepsCompleted, translate } = this.props;
+		const steps = [ STEPS.CONTACT_FORM, STEPS.BUSINESS_ADDRESS, STEPS.STATS ];
```

Comment:
Do we even need the whitelist? The worst thing that would happen if we dropped this check is that users would see 'Next' instead of 'Skip Link' on a previously completed step when they return back to it. cc @rickybanister

Reply:
Good call, it makes sense to apply this to all completed steps, done in https://github.com/Automattic/wp-calypso/commit/68214295ebbaf912fc366ee351405a449bc45184.